### ISSUE TITLE
Font is not displayed in Firefox

### ISSUE THREAD

**Issue by atelierbram** [RedHatBrand/Overpass#27]
Firefox console gives this message (Overpass-Light, for .woff and .ttf): 

``` bash
downloadable font: 
Layout: DFLT table doesn't satisfy the spec. for script tag DFLT
GPOS: Failed to parse script list table
Failed to parse script list table
rejected by sanitizer
```

Maybe due to [stricter validation rules](https://bugzilla.mozilla.org/show_bug.cgi?id=1244693)?

**Comment by davelab6** [comment]
https://github.com/khaledhosny/ots/blob/master/docs/HowToFix.md#dflt-script-doesnt-satisfy-the-spec explains

**Comment by DelveFonts** [comment]
Thank you @atelierbram for bringing this to our attention. This particular issue will be resolved with the next update (currently in progress) as the project was migrated to the Glyphs app for production.  /cc @davelab6

**Comment by davelab6** [comment]
Nice! Would be great to try the https://github.com/googlei18n/fontmake
glyphs compiler :)

**Comment by funtoosh** [comment]
So this has not been fixed yet? :-(
Quite a massive issue, imho ...

**Comment by andyfitz** [comment]
It's been fixed,  version 3.0 is out !
